src/tcgwars/logic/impl/gen3/PopSeries2.groovy

```diff
@@ -362,6 +427,7 @@ public enum PopSeries2 implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 30
+            flip 1, {}, { directDamage 10, self}
```

Comment:
This Pikachu is a reprint of the Base Set one

`return copy(BaseSetNG.PIKACHU, this);`